Sources/SwiftSyntax/IncrementalParseTransition.swift

```diff
@@ -206,6 +206,8 @@ public struct IncrementalParseLookup {
   fileprivate let transition: IncrementalParseTransition
   fileprivate var cursor: SyntaxCursor
 
+  /// Create a new `IncrementalParseLookup` that can look nodes up based on the
+  /// given `IncrementalParseTransition`.
```

Comment:
Would it make sense to add double ticks. At least around `IncrementalParseTransition`?